lms/djangoapps/instructor/views/data_access_constants.py

```diff
@@ -49,7 +49,7 @@ class Inclusion:
                          'F': Inclusion.FILTER}
 
 INCLUDE_SECTION_PATTERN = re.compile('chapter|sequential')
-INCLUDE_PROBLEM_PATTERN = re.compile('problem')
+INCLUDE_PROBLEM_PATTERN = re.compile('problem|openassessment|submit-and-compare|freetextresponse|xblockmufi|qualtricssurvey|inline-dropdown|poll|image-coding')
```

Comment:
As an example, look for INLINE_ANALYTICS_SUPPORTED_TYPES in lms.env.common and lms.env.aws and follow a similar pattern.